### PR TITLE
removed account selector dropdown for non loggedin users

### PR DESCRIPTION
**Pull request by jagathks**
As title says removed account selector dropdown for non loggedin user's.
For me it doesn't make sense to show dropdown for all users. So here is my change.
image

### CODE REVIEW

**Comment by harpreet9629** [comment]
You can convert Log In link with a Button with the Same Style. I think it will be good idea :)

**Comment by jagathks** [comment]
Something like this?
image

**Comment by harpreet9629** [comment]
What if Remove Login on the left side? Because there two buttons in right side :)

**Comment by jagathks** [comment]
@lazynina @maebeam any suggestion on this?
@harpreet9629  both the button resolve to same function on right side bar. may be we can simplify as Login/Signup.

**Not relevant**, but i don't know where else to post:
Any one can point me in right direction/help me on how to setup/start develop on backend/core with Goland IDE?
Since I am new to Go lang, i have spent most my development time in Java(InteliJ, STS), Python(PyCharm), Nodejs, Typescript i need some kik-start in IDE setup/configuration w.r.t. backend/core.

**Comment by cat-ico** [comment]
@jagathks 
> Any one can point me in right direction/help me on how to setup/start develop on backend/core with Goland IDE?

Here's the basic setup guide https://docs.bitclout.com/code/dev-setup
If you have some specific problems, then try Bitclout pulse discord https://discord.gg/MqbbTpcs - #-technical-discussion channel

**Comment by tijno** [comment]
not everyone looks in the same place for login buttons - so having some dupplication of login link left and right i dont think is a bad thing.

**Comment by jagathks** [comment]
@tijno agreed, but left and right sections are common to all pages for pre and post login's.
so will it make any difference if er keep in one place? also thinking interms of UI/UX.

**Review by maebeam — APPROVED** [review]
Looks good to me, will wait for @lazynina's approval to merge

**Comment by lazynina** [comment]
> @tijno agreed, but left and right sections are common to all pages for pre and post login's.
> so will it make any difference if er keep in one place? also thinking interms of UI/UX.

Right bar doesn't appear on mobile, so we need to keep it in the left bar for now.

**Review by lazynina — APPROVED** [review]

**Pull request merged**
